ispdb/protonmail.com.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<clientConfig version="1.1">
+  <emailProvider id="protonmail.com">
+    <domain>protonmail.com</domain>
+    <domain>protonmail.ch</domain>
+    <domain>pm.me</domain>
+
+    <displayName>ProtonMail</displayName>
+    <displayShortName>ProtonMail</displayShortName>
+
+    <incomingServer type="imap">
+      <hostname>127.0.0.1</hostname>
+      <port>1143</port>
+      <socketType>STARTTLS</socketType>
+      <username>%EMAILADDRESS%</username>
+    </incomingServer>
+
+    <outgoingServer type="smtp">
+       <hostname>127.0.0.1</hostname>
+       <port>1025</port>
+       <socketType>STARTTLS</socketType>
+       <username>%EMAILADDRESS%</username>
+    </outgoingServer>
+
+    <enable visiturl="https://protonmail.com/bridge/">
+      <instruction>You must first install and activate the ProtonMail Bridge.</instruction>
+    </enable>
+
```

Comment:
Alternatively, Thunderbird account config UI could implement showing this `<enable>` message, with a button to open the URL in a browser. That would lead the users the right way.

Thank you for adding this information to the config!